scene/2d/collision_polygon_2d.cpp

```diff
@@ -159,6 +159,7 @@ void CollisionPolygon2D::_notification(int p_what) {
 			}
 #else
 			draw_colored_polygon(polygon, get_tree()->get_debug_collisions_color());
+			draw_polyline(polygon, get_tree()->get_debug_collisions_color());
```

Comment:
Missed `draw_line` here as well?

Reply:
Is this part used at all? I noticed CollisionPolygon2Ds appear in a different color and already have an outline anyway.

Reply:
If `DEBUG_DECOMPOSE` is configurable then it might make sense, else this could be handled by `ProjectSettings` instead?

Reply:
This code is used in non-tool debug to display a unique shape instead of the decomposed convex shapes.

Adding `draw_polyline` and `draw_line` here would draw an extra line on top of the red line in this case, so this block shouldn't change.